ml_service/pipelines/deploy-model-to-appservice.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent { label 'master' }
+    environment {
+        ML_IMAGE_FOLDER         = 'imagefiles'
+        IMAGE_NAME              = 'mlmodelimage'
+        MODEL_NAME = 'diabetes_regression_model.pkl'
+        MODEL_VERSION = "${MODEL_VERSION}"
+        SCORE_SCRIPT = 'scoring/score.py'
+        RESOURCE_GROUP = "${RESOURCE_GROUP}"
+        WORKSPACE_NAME = "${WORKSPACE_NAME}"
+        PACKAGE_FOLDER = './download'
+    }
+    stages {
+        stage('initialize') {
+            steps {
+                echo 'Remove the previous one!'
+            }
+            post {
+                always {
+                    deleteDir() /* clean up our workspace */
+                }
+            }
+        }
+        stage('download_model') {
+            steps {
+                echo "Downloading..."
+                /*checkout scm*/
+                checkout([$class: 'GitSCM', branches: [[name: '*/ml_model_uc81']],
```

Comment:
Might want to change the branch name to master here (esp. if you're planning to delete this branch after merging)

Reply:
Hey Toan, I'm thinking the checkout branch name should be master? Since after this is merged, it should point to master

Reply:
Ah sorry ignore me, I was looking at the outdated file. Oops.